src/World.h

```diff
@@ -142,6 +142,30 @@ class cWorld :
 		std::vector<Vector3i> m_SendQueue;
 	};
 
+	class cTaskPlaySound :
```

Comment:
cWorld does not seem the right place for this. If it really is generic it should be in a seperate file. However I think we should make cWorld::ScheduleTask support lambdas and then this would be a trivial task.